src/useLockBodyScroll.ts

```diff
@@ -15,15 +15,33 @@ export function getClosestBody(el: Element | HTMLElement | HTMLIFrameElement | n
   return getClosestBody((el as HTMLElement).offsetParent!);
 }
 
+function preventDefault(rawEvent: TouchEvent): boolean {
+  const e = rawEvent || window.event;
+  // Do not prevent if the event has more than one touch (usually meaning this is a multi touch gesture like pinch to zoom).
+  if (e.touches.length > 1) return true;
+
+  if (e.preventDefault) e.preventDefault();
+
+  return false;
+}
+
 export interface BodyInfoItem {
   counter: number;
   initialOverflow: CSSStyleDeclaration['overflow'];
 }
 
+const isIosDevice =
```

Comment:
Let me know if there's already a helper to detect this

Reply:
No, we don't have one.